### DISCUSSION TITLE
Idea  How to make a test tone with 1kHz on I2S to an external DAC

### DISCUSSION THREAD

**Discussion by Micha2007fe** [schreibfaul1/ESP32-audioI2S#455] [Q&A]
HI,
has anybody a idea, how to make a Test Tone with this lib and and ESP32 and a extrenal DAC ?
I would like to have for example 1 Khz Testtone.
But I do not want to use a SDCard, so ideal direct generated from ESP32 over I2S to the external DAC.
Kind Regards

**Comment by MartinNohr** [comment]
I'd like to do the same thing. Any ideas yet?
